pkg/scsi/spc.go

```diff
@@ -390,7 +392,9 @@ func SPCStartStop(host int, cmd *api.SCSICommand) api.SAMStat {
 		return api.SAMStatReservationConflict
 	}
 
-	cmd.InSDBBuffer.Resid = 0
+	if cmd.InSDBBuffer != nil {
```

Comment:
this doesn't seem to be required. I'm reviewing from the angle that there are cases where R_bit is NOT set in some cases where as R_bit will be set for that opcode generally. Code doesn't seem to handle that some cases.

Reply:
Can we still have a check over here. A bit doubtful on what flags client might send with this command.